experimenter/experimenter/kinto/tasks.py

```diff
@@ -264,6 +265,9 @@ def nimbus_push_experiment_to_kinto(collection, experiment_id):
 
         kinto_client = KintoClient(collection)
 
+        if experiment.published_date is None:
+            experiment.published_date = timezone.now()
```

Comment:
Actually we could always force overwrite this, if for whatever reason it's still set when we get here and we're publishing to kinto, then the most recent publish timestamp would be the canonical one wouldn't it?

Reply:
I don't necessarily know that we'd want that? With rollouts that increment their pop size they would be moving their published_date pretty frequently and I imagine we'd want that to stay static, wouldn't we?

Reply:
Oh yes very good pointtttttt......... Then it's more about state ie if it's not live we'd always want to overwrite?

Reply:
Perhaps? My understanding was it wasn't meant to roll but if we want it to roll then yes what you've described makes sense

Reply:
Yeah like I'd imagine that date is the most recent date that a launching recipe is pushed to kinto.  We can unset it on timeout/rejection but those tasks aren't gauranteed to run/complete so if we got here, we're not live, and for whatever reason publish date is set, wouldn't we always want the most recent date?

Reply:
Fair enough — okay so switching it to
- set if not set
- set if currently live

Reply:
* if currently *not* live, right?

Reply:
Ah yes sorry